Jenkinsfile

```diff
@@ -690,6 +690,11 @@ pipeline {
                         }
                     } // steps
                 } // stage ('Commit Pragma tests')
+                stage('Self Unit Test') {
+                    steps {
+                        selfUnitTest()
```

Comment:
Note that as suspected, this still requires the `@Library` self-reference to run [any new code (in the future) in] the `selfUnitTest()` function.  Sadly.